app/assets/stylesheets/layout.scss

```diff
@@ -1063,6 +1063,17 @@ form {
     line-height: $line-height;
   }
 
+  .actions {
+    label {
+      margin-left: auto;
+      margin-right: auto;
+    }
```

Comment:
How about moving this code to the `public-form` mixin, which contains the rest ofthe styles for the `.actions` selector?